### PR TITLE
Add Frameworks Week for Applied AI #6

### PR DESCRIPTION
**Pull request by urasmutlu**
# Add Frameworks Week for Applied AI #6

- Modified the TF2 and Keras notebooks, added markdown explanations to them
- Added a new notebook for PyTorch CNN MNIST example

### CODE REVIEW

**Comment by ahmetmeleq** [comment]
1. In the first notebook:
`plt.imshow(mnist_training_images[0] ,cmap = 'gray')`
does not work for me, instead, this works:
`plt.imshow(mnist_training_images[0][:,:,0] ,cmap = 'gray')`

2. same for:
`plt.imshow(mnist_test_images[0] ,cmap = 'gray')`

3. same for:
`plt.imshow(tf.reshape(batch_images[0], shape=(img_width, img_height, 1)) ,cmap = 'gray')`

**Comment by ahmetmeleq** [comment]
We can add this to the end, as markdown, in all notebooks:

Bonus - Try to: 

- get a test image
- plot the image
- make a model prediction on the image 
- print the predicted label and the actual label!

(I later saw that we already have this part for the last notebook. We can also map class labels to their corresponding strings)

**Comment by ahmetmeleq** [comment]
Markdown parts and everything else look perfect, eline sağlık

**Comment by EralpKocas** [comment]
As we have already discussed in inzva, markdowns and code are fine. I did not try to run the codes but they were working for me in the previous batch. 

Two suggestions from me would be the following: we can add an explanatory text for the task introduced and an example use case for making people following the notebook familiar with it. Also, we can add a couple of notes about frameworks model building logic. For example, in Keras Functional, we build models as DAGs. It might help people to understand the frameworks better. Your PyTorch note is a great example of it: why we don't need backward function.

Other than these two points, eline sağlık. All seems well :)

**Comment by urasmutlu** [comment]
Thanks @EralpKocas and @ahmetmeleq for your reviews. Updated the notebooks and merging this PR with master
